quickstatements/latest/config.json

```diff
@@ -0,0 +1,18 @@
+{
+	"site" : "${MW_SITE_NAME}" ,
+	"bot_config_file" : "/var/www/html/bot.ini" ,
+	"logfile" : "/var/log/quickstatements/tool.log" ,
+	"sites" : {
+		"${MW_SITE_NAME}" : {
+			"oauth" : { "language":"${MW_SITE_LANG}" , "project":"${MW_SITE_NAME}" , "ini_file":"/var/www/html/quickstatements/oauth.ini" , "publicMwOAuthUrl":"http://${WB_PUBLIC_HOST_AND_PORT}/w/index.php?title=Special:OAuth" , "mwOAuthUrl":"http://${WIKIBASE_HOST}/w/index.php?title=Special:OAuth" , "mwOAuthIW":"mw" } ,
+			"server" : "${WB_PUBLIC_HOST_AND_PORT}" ,
+			"api" : "http://${WIKIBASE_HOST}/w/api.php" ,
```

Comment:
We probably don't want to hard code http here, this should probably be part of the env var.

(same below)